console/console-init/ui/src/modules/iot-certificates/components/CertificateForm/CertificateForm.tsx

```diff
@@ -71,6 +71,7 @@ export const CertificateForm: React.FunctionComponent<ICertificateFormProps> = (
     value: string | boolean,
     event: React.FormEvent<HTMLInputElement>
   ) => {
+    // const cerftificateField = event?.target?.name;
     const { name: cerftificateField } = event.currentTarget;
     const newCertificateData = { ...certificateFormData };
     (newCertificateData as any)[cerftificateField] = value;
```

Comment:
Does this need to be an 'any'?

Reply:
@rkpattnaik780 Please take a look into this

Reply:
Yes this is supposed to be any.